### PR TITLE
Overlay plots and template slides

### PR DESCRIPTION
**Pull request by EmyrClement**
Hi Louise,

Here are my changes for the overlay plots and template slides.  I've added a readme, but let me know if anything isn't obvious.  The scripts should still all work when some files are missing.  One thing I should add is a list of samples (relVals) that I've assumed are being run over (ttbar and electron/muon guns at each PU).

I also modified L1TrackNtuplePlot.C to include the name of the tree in the input file as an argument.  I found this necessary recently with the hybrid algorithms, as I would run the ntuple maker over several different track collections in the same job, and the output tree names would be different.  I can remove it if you prefer, though it's an empty string by default, so hopefully isn't too much of an issue.  Alternatively, I can put it at the end of the list of arguments.

One other change, I reduced the number of bins in the efficiency vs pt plots (or rebinned by a large amount), as the relVals don't have as much statistics as we used to have.

Thanks,
Emyr

### CODE REVIEW

**Comment by EmyrClement** [comment]
Another comment - the choice of plots in the slides is different to what was in the template slides from the review.  I just removed/added some to reflect the samples we have available to us.

**Comment by skinnari** [comment]
Thanks @EmyrClement ! Sounds great, I'll merge this over the weekend.

**Comment by EmyrClement** [comment]
I've added the list of samples to the README

File: L1Trigger/TrackFindingTracklet/test/L1TrackNtuplePlot.C [hunk 1]
```diff
@@ -93,7 +93,7 @@ void L1TrackNtuplePlot(TString type, int TP_select_injet=0, int TP_select_pdgid=
 
   // ----------------------------------------------------------------------------------------------------------------
   // read ntuples
-  TChain* tree = new TChain("L1TrackNtuple/eventTree");
+  TChain* tree = new TChain("L1TrackNtuple"+treeName+"/eventTree");
```
Comment (skinnari) [inline L1Trigger/TrackFindingTracklet/test/L1TrackNtuplePlot.C:96]: @EmyrClement if we add this, we also need to modify the ntuple maker, no?

**Review by skinnari — APPROVED** [review]

**Pull request merged**